packages/story-editor/src/components/mediaRecording/footer.js

```diff
@@ -301,18 +357,25 @@ function Footer({ captureImage, videoRef }) {
               {__('Stop Recording', 'web-stories')}
             </StopButton>
           )}
-          {!isRecording && !isCountingDown && (
-            <>
+          {!isRecording &&
+            !isCountingDown &&
+            (hasVideo ? (
+              <>
+                <RecordingButton onClick={onStart}>
+                  <Icons.Camera width={24} height={24} aria-hidden />
+                  {__('Record Video', 'web-stories')}
+                </RecordingButton>
+                <CaptureButton onClick={onCapture}>
+                  <Icons.PhotoCamera width={24} height={24} aria-hidden />
+                  {__('Take a photo', 'web-stories')}
+                </CaptureButton>
+              </>
+            ) : (
               <RecordingButton onClick={onStart}>
-                <Icons.Camera width={24} height={24} aria-hidden />
-                {__('Record Video', 'web-stories')}
+                <Icons.Mic width={24} height={24} aria-hidden />
+                {__('Record Audio', 'web-stories')}
```

Comment:
Hm, audio can be recorded only when the user first disables the video from the menu bar? A bit hard to discover but might be just me :)

Reply:
We can do `[ Record Audio (Only) ]` button, WDYT @swissspidy ?

Reply:
Let's handle UX/UI improvements like that in a follow-up ticket, shouldn't hold off this PR.